cmd/client/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/jeka2708/golang-training-enterprise-grpc/pkg/api"
+	"google.golang.org/grpc"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	var serverAddr = os.Getenv("backend")
```

Comment:
please decide what you prefer, uppercase or lowercase for env vars and follow the naming convention